dist/index.html

```diff
@@ -24,25 +24,25 @@ <h1 class='current-recipe-name'></h1>
       <section class="three-images bottom-left-main">
           <div class="left-random-card card">
             <img class="left-random-image random-image" img src="https://spoonacular.com/recipeImages/595736-556x370.jpg"/>
-            <h1 class="left-random-name"></h1>
+            <h1 class="left random-name"></h1>
```

Comment:
these changes to the class name are why the names are not showing up, these classes are tied to some of the javascript functionality